File: tests/fixtures/sample.py
"""Sample Python module for testing."""


def sample_function(input_str: str) -> str:
    """Sample function that processes a string.

    Args:
        input_str: Input string to process

    Returns:
        Processed string
    """
    if not input_str:
        return "empty"
    return f"value: {input_str}"


class SampleClass:
    """Sample class for testing."""

    def __init__(self, name: str, value: int):
        """Initialize SampleClass.

        Args:
            name: Name value
            value: Integer value
        """
        self.name = name
        self.value = value

    def get_description(self) -> str:
        """Get description.

        Returns:
            Description string
        """
        return f"{self.name}: {self.value}"

File: tests/fixtures/sample.js
/**
 * Sample JavaScript module for testing.
 */

/**
 * Sample function that processes a string.
 * @param {string} inputStr - Input string to process
 * @returns {string} Processed string
 */
function sampleFunction(inputStr) {
  if (!inputStr) {
    return 'empty';
  }
  return `value: ${inputStr}`;
}

/**
 * Sample class for testing.
 */
class SampleClass {
  /**
   * Create a SampleClass.
   * @param {string} name - Name value
   * @param {number} value - Integer value
   */
  constructor(name, value) {
    this.name = name;
    this.value = value;
  }

  /**
   * Get description.
   * @returns {string} Description string
   */
  getDescription() {
    return `${this.name}: ${this.value}`;
  }
}

module.exports = {
  sampleFunction,
  SampleClass,
};
